spec/requests/dogs_spec.rb

```diff
@@ -6,7 +6,7 @@
       Dog.create(
         name: 'Harold',
         age: 4,
-        enjoys: 'barking',
+        enjoys: 'benjoys running in the yard',
```

Comment:
Only because of the other changes I see...looks like enjoys was misspelled. Very minor. If it is concerning to you. Since this is in an isolated testing environment, no need to correct.